src/pages/App.js

```diff
@@ -47,8 +49,17 @@ const App = () => {
             </Suspense>
           }
         />
+        <Route
+          path="/books"
```

Comment:
for paths you can create const and then use it for better accessibility and avoiding typo